capstone/src/main/java/com/google/univiz/api/SearchData.java

```diff
@@ -1,4 +1,12 @@
 package com.google.univiz.api;
 
 /** SearchData represents the college data specific to the Search feature */
-public class SearchData {}
```

Comment:
i think we want to use ```AutoValue``` for the ```SearchData``` and ```SuggestionData``` class. look at #37 for reference on how to use it, and if you need some help, put it in the chat. also #52 has another ```AutoValue``` example.

Reply:
ah will do!